CoreScriptsRoot/Modules/Stats/BarGraph.lua

```diff
@@ -0,0 +1,62 @@
+--[[
+		Filename: BarGraph.lua
+		Written by: dbanks
+		Description: A simple bar graph.
+--]]
+
+--[[ Services ]]--
+local CoreGuiService = game:GetService('CoreGui')
+
+--[[ Globals ]]--
+local BarColor = Color3.new(0.1, 0.7, 0.1)
+
+--[[ Modules ]]--
+local folder = CoreGuiService:WaitForChild("RobloxGui")
```

Comment:
Might be late to the party on this one as I think we call this folder in a lot of other places, but a name like StatsFolder may be more descriptive